Update bi migration tool docs (#135)
wso2/docs-ballerina-integrator: en/docs/developer-guides/tools/migration-tools/mulesoft-migration-tool.md

# MuleSoft Migration Tool

> This guide explains how to use the integrated migration feature in WSO2 Integrator: BI to convert [MuleSoft](https://www.mulesoft.com) applications into Ballerina packages.

Reviewer: How about
```suggestion
This guide explains how to convert existing [MuleSoft](https://www.mulesoft.com) applications into integrations compatible with the [WSO2 Integrator: BI](https://wso2.com/integrator/bi/).
```
?

> ## Tool overview

Reviewer: can we just say
```suggestion
## Overview
```
?

> The migration tool is now integrated directly into the WSO2 Integrator: BI, providing a user-friendly wizard interface for converting MuleSoft projects. The tool accepts either a MuleSoft project directory or a standalone Mule `.xml` configuration file as input. It generates equivalent Ballerina packages that can be opened directly in WSO2 Integrator: BI.

Reviewer: ```suggestion
This migration support is directly integrated into the WSO2 Integrator: BI, providing a user-friendly wizard interface for converting MuleSoft projects. The tool accepts either a MuleSoft project directory or a standalone Mule `.xml` configuration file as input and, generates equivalent Ballerina packages that can be opened directly in WSO2 Integrator: BI.
```
